packages/react/src/Pagination/Pagination.tsx

```diff
@@ -124,30 +124,21 @@ export const Pagination = forwardRef(
     }
 
     return (
-      <nav {...restProps} className={clsx('ams-pagination', className)} ref={ref} aria-label="Paginering">
+      <nav {...restProps} className={clsx('ams-pagination', className)} ref={ref}>
+        <span className="ams-visually-hidden">Paginering</span>
```

Comment:
Lets not hardcode dutch in here. There has been an issue from nlds asking us to your props instead of hardcoded strings.

Reply:
See #1220

Reply:
Adjusted.